package.json

```diff
@@ -14,11 +14,12 @@
     "prebuild": "npm run clean",
     "build": "tsc -p tsconfig.dist.json"
   },
+  "dependencies": {
+    "graphql": "^14.0.2",
+    "rxjs": "^6.0.0"
```

Comment:
It looks like this is supposed to be a dev dependency, since a peer dependency is defined

Reply:
my understanding is that if this package is actually using it, it should be a dependency, am I wrong? 🤔

Reply:
Not exactly. My minimal litmus test is that if consuming apps are expected to be exposed to a dependency, it can be a peer. If rxjs were a private dependency, it should be defined here. reactive-graphql consumers are definitely expected to also be rxjs consumers. Tests import and use so I would expect it to be a devDependency as well.